openapi/openapi.yaml

```diff
@@ -42,6 +42,7 @@ paths:
               schema:
                 $ref: '#/components/schemas/Error'
       parameters:
+      - $ref: '#/components/parameters/name'
```

Comment:
we may not need to introduce this parameter, it seems we can get resource with search parameter, e.g.
`/api/maestro/v1/resources?search=name = 'cluster3'`

Reply:
yes. search can be used for any field name. But it seems `/api/maestro/v1/resources?name=cluster3` is widely used.

Reply:
@markturansky How do you use in your side?